fili-core/src/main/java/com/yahoo/bard/webservice/table/availability/Availability.java

```diff
@@ -22,33 +22,21 @@
      *
      * @return A set of names for datasources backing this table
      */
-    SortedSet<TableName> getDataSourceNames();
-
-    /**
-     * The availability for a given column.
-     *
-     * @param c  A column
-     *
-     * @return The list of intervals that column is available for.
-     */
-    List<Interval> get(Column c);
+    Set<TableName> getDataSourceNames();
 
     /**
      * The availability of all columns.
      *
      * @return The intervals, by column, available.
      */
-    Map<Column, List<Interval>> getAvailableIntervals();
+    Map<Column, Set<Interval>> getAllAvailableIntervals();
```

Comment:
The question I was thinking is why is `SimplifiedIntervalList` not `SimplifiedIntervalSet`